docs/advanced/append-nodes.md

```diff
@@ -0,0 +1,50 @@
+# Append additional nodes to the builder
+
+You can append nodes to the builder that is going to be created with the
+`append` input in the form of a YAML string document to remove limitations
+intrinsically linked to GitHub Actions (only string format is handled in the
+input fields):
+
+| Name              | Type   | Description                                                                                                                                                                       |
+|-------------------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`            | String | [Name of the node](https://docs.docker.com/engine/reference/commandline/buildx_create/#node). If empty, it is the name of the builder it belongs to, with an index number suffix. |
```

Comment:
iiuc this is only needed when user wants to modify/remove the node later. If this is true then better to mention it here to not confuse users about why they would care about this field.